Creating posts on OLS's annual report: it's broken into 3 parts (#202)
open-life-science/open-life-science.github.io: _posts/2020-10-01-annual-report-part-2.md

In OLS-2, a total of 32 projects by a group of 52 mentees were selected, which is a ~50% increase in project numbers and participants over the previous cohort. These project leads will be trained and mentored between September and December of 2020 with the support of 32 mentors and 67 members in the expert community. These members represent several countries from 6 continents and will advance the knowledge of Open Science in their communities. 6 projects from [*The Turing Way*](https://the-turing-way.netlify.app/welcome) affiliated with [The Alan Turing Institute](https://www.turing.ac.uk/) joined the project in the second cohort, as part of a [collaboration](/ols-2#collaborators). Through this collaboration, the international community of *The Turing Way* will be able to access mentoring and training support through OLS. Read more details about this cohort in [this blog post]({% link _posts/2020-09-01-ols2-announcement.md %}).

> We continue to invite feedback from our participants to ensure that they feel adequately supported and find their experience of working with OLS mutually rewarding. We encourage participants to develop their projects on their GitHub repositories and share with others in the program through a centralised GitHub-based issue tracker. They explore and discuss each other's projects and share feedbacks on [OLS-2 GitHub issues](https://github.com/open-life-science/ols-2/issues) and shared HackMds. They also volunteer to host coworking sessions to work and build connections with others in the program. We have pathways for our graduates to return as mentors and experts (many OLS-1 graduates have returned to OLS-2) and continue to gain support from the OLS even after they have finished their training.

Reviewer: ```suggestion
We continue to invite feedback from our participants to ensure that they feel adequately supported and find their experience of working with OLS mutually rewarding. We encourage participants to develop their projects on their GitHub repositories and share with others in the program through a centralised GitHub-based issue tracker. They explore and discuss each other's projects and share feedback on [OLS-2 GitHub issues](https://github.com/open-life-science/ols-2/issues) and shared HackMDs. They also volunteer to host coworking sessions to work and build connections with others in the program. We have pathways for our graduates to return as mentors and experts (many OLS-1 graduates have returned to OLS-2) and continue to gain support from the OLS even after they have finished their training.
```

## Cohort expense and cost - and its shoestring budget 

Our website helps to give us an incredibly professional face thanks to Bérénice’s tireless and utterly incredible efforts (including during her maternity leave). But, as shiny as our site might look, we’re still operating on a shoestring budget - most of the preparation work we do is out of hours and on our own time. Likewise, OLS mentors and experts are all currently unpaid. 

> Estimated real cost for each cohort is GBP £24,259 in founder, mentor, and expert time, as well as some infrastructure costs. Ideally, we would like to meet these costs - as such it probably comes as no surprise that the OLS team is currently actively seeking funding and applying for external fund sources.

Reviewer: ```suggestion
The estimated real cost for each cohort is GBP £24,259 in founder, mentor, and expert time, as well as some infrastructure costs. Ideally, we would like to meet these costs - as such it probably comes as no surprise that the OLS team is currently actively seeking funding and applying for external fund sources.
```

<iframe src="https://docs.google.com/spreadsheets/d/1_Q8atU9Xsh5aXQuQpxizyMUdKWug_jYgMobzhxxPE8c/pubhtml?widget=true&amp;headers=false" style="width: 100%; height: 400px"></iframe>

> Many of the expenses required to run this project effectively have come from the founders' own pockets. Some funding resources can be reached with requests for support to sympathetic organisations, however claiming some of our expenses can sometimes be challenging -- not all items can be funded due to funding organisation's policies, and administrative formalities and/or blog posts are required for each funded item. Whilst running a non-profit organisation outside of our day jobs, we've unfortunately become extremely time-poor to constantly look for small funds or offer invisible labour associated with funding obligations.

Reviewer: ```suggestion
Many of the expenses required to run this project effectively have come from the founders' own pockets. Some funding resources can be reached with requests for support to sympathetic organisations, however claiming some of our expenses can sometimes be challenging -- not all items can be funded due to funding organisation's policies, and administrative formalities and/or blog posts are required for each funded item. Whilst running a non-profit organisation outside of our day jobs, we've unfortunately become extremely time-poor and struggle to constantly look for small funds or offer invisible labour associated with funding obligations.
```

> ### Software Sustainability Institute partnership for OLS-2

Author: ```suggestion
### Software Sustainability Institute fellowship to support OLS-2
```

---
layout: post
title: OLS Project and Community report - Part 2
authors:
- bebatut
- malvikasharan
- yochannah
image: https://images.unsplash.com/photo-1523540939399-141cbff6a8d7?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80
photos:
  name: Sam Dan Truong, Unsplash id: @sam_truong
  license: CC-BY
  url: https://unsplash.com
---

# Year one of Open Life Science (OLS) - Part 2

*This is the second part of the 3 parts OLS project and community report: **How far we have come***

> ## OLS-1 to OLS-2: *cohort doubled in size*

Author: In the previous part, we discussed OLS-1. But at this point, I am happy to actually merge all these broken parts into one file as suggested by you so that this can go under the OLS-1 section.

## Cohort expense and cost - and its shoestring budget 

Our website helps to give us an incredibly professional face thanks to Bérénice’s tireless and utterly incredible efforts (including during her maternity leave). But, as shiny as our site might look, we’re still operating on a shoestring budget - most of the preparation work we do is out of hours and on our own time. Likewise, OLS mentors and experts are all currently unpaid. 

Estimated real cost for each cohort is GBP £24,259 in founder, mentor, and expert time, as well as some infrastructure costs. Ideally, we would like to meet these costs - as such it probably comes as no surprise that the OLS team is currently actively seeking funding and applying for external fund sources.

> <iframe src="https://docs.google.com/spreadsheets/d/1_Q8atU9Xsh5aXQuQpxizyMUdKWug_jYgMobzhxxPE8c/pubhtml?widget=true&amp;headers=false"></iframe>

Reviewer 2: ```suggestion
<iframe src="https://docs.google.com/spreadsheets/d/1_Q8atU9Xsh5aXQuQpxizyMUdKWug_jYgMobzhxxPE8c/pubhtml?widget=true&amp;headers=false" style="width: 100%; height: 400px"></iframe>
```

> *OLS real cost for one cohort (financial requirements are currently not met)*

Reviewer 2: ```suggestion
```

Reviewer 2: It is already displayed in the iframe

> Many of the expenses required to run this project effectively have come from the founders’ own pockets. Some funding resources can be reached with requests, however claiming some of our expenses can sometimes be challenging -- not all items can be funded due to funding organisation's policies, and administrative formalities and/or blog posts are required for each funded item. Whilst running a non-profit organisation outside of our day jobs, we’ve unfortunately become extremely time-poor to constantly look for small funds or offer invisible labor associated with funding obligations.

Reviewer: ```suggestion
Many of the expenses required to run this project effectively have come from the founders’ own pockets. Some funding resources can be reached with requests for support to sympathetc organisations, however claiming some of our expenses can sometimes be challenging -- not all items can be funded due to funding organisation's policies, and administrative formalities and/or blog posts are required for each funded item. Whilst running a non-profit organisation outside of our day jobs, we’ve unfortunately become extremely time-poor to constantly look for small funds or offer invisible labor associated with funding obligations.
```

> Many of the expenses required to run this project effectively have come from the founders’ own pockets. Some funding resources can be reached with requests for support to sympathetic organisations, however claiming some of our expenses can sometimes be challenging -- not all items can be funded due to funding organisation's policies, and administrative formalities and/or blog posts are required for each funded item. Whilst running a non-profit organisation outside of our day jobs, we’ve unfortunately become extremely time-poor to constantly look for small funds or offer invisible labour associated with funding obligations.

Reviewer 2: ```suggestion
Many of the expenses required to run this project effectively have come from the founders' own pockets. Some funding resources can be reached with requests for support to sympathetic organisations, however claiming some of our expenses can sometimes be challenging -- not all items can be funded due to funding organisation's policies, and administrative formalities and/or blog posts are required for each funded item. Whilst running a non-profit organisation outside of our day jobs, we've unfortunately become extremely time-poor to constantly look for small funds or offer invisible labour associated with funding obligations.
```

### Software Sustainability Institute partnership for OLS-2

> Two co-founders are fellows of the Software Sustainability Institute (UK), which provided small fellowship funds with which we manage to support the communication and application management platforms required for OLS-2 (EasyChair, Zoom, professional mentor training etc.). We are grateful to receive personal and organisational support in redirecting this fellowship towards OLS infrastructure needs. 

Reviewer 2: ```suggestion
Two co-founders are fellows of the [Software Sustainability Institute (UK)](https://www.software.ac.uk/), which provided small fellowship funds with which we manage to support the communication and application management platforms required for OLS-2 (EasyChair, Zoom, professional mentor training etc.). We are grateful to receive personal and organisational support in redirecting this fellowship towards OLS infrastructure needs. 
```

> ### EOSC-Life Training grant to partially support OLS-3

Reviewer 2: ```suggestion
### EOSC-Life Training grant for OLS-3
```

Reviewer 2: It feels otherwise than SSI may have cover all costs for OLS-2

> For one cohort in 2021, OLS has been awarded EOSC-Life (https://www.eosc-life.eu/) open call training funding of up to 20,000 Euros to develop a special call to train and mentor up to 15 projects from within the EOSC-Life research infrastructure. This will partially cover costs for OLS-3, which will be designed over the next few months by including a new curriculum.

Reviewer 2: ```suggestion
For one cohort in 2021, OLS has been awarded [EOSC-Life](https://www.eosc-life.eu/) open call training funding of up to 20,000€ to develop a special call to train and mentor up to 15 projects from within the EOSC-Life research infrastructure. This will partially cover costs for OLS-3, which will be designed over the next few months by including a new curriculum.
```

This opportunity will help widen the accessibility of our service and resources for the EOSC-Life community while we continue to support our participants from developing countries. Thanks to Neil Chue Hong, Bastian Greshake Tzovaras, Mateusz Kuzak, Andrew Stewart, Naomi Penfold, Fotis E. Psomopoulos, and Emmy Tsang for providing references and generous testimonials about the program. You can find a copy of our grant application  on [Zenodo](https://zenodo.org/record/4060367).

> ***Please see the [Part 1 of this report](./2020-10-01-annual-report-part-1) and [Part 3 of this report](./2020-10-01-annual-report-part-3)***

Reviewer 2: ```suggestion
***TO BE CONTINUED**: You can now read last part of the report: [**Where we want to go**]({% link _posts/2020-10-01-annual-report-part-3.md %}). If you missed it, you can also read the first part: [**Where we started**]({% link _posts/2020-10-01-annual-report-part-1.md %})*
```
